### ISSUE TITLE
🤖 Android: Unsupported class file major version 61 on React Native 0.67.5

### ISSUE THREAD

**Issue by betanyeli** [software-mansion-labs/react-native-turbo-demo#212]
### Description

I'm encountering a persistent build failure when trying to integrate react-native-turbo into my React Native CLI project. The error message consistently indicates an **unsupported class file major version**, despite attempts to adjust Java, Gradle, and Android Gradle Plugin versions.

Project Environment:

- React Native version: 0.67.5
- Gradle versions tried: [7.3.3, 8.4, 7.0.0]
- Android Gradle Plugin versions tried: [4.2.2, 7.3.3]
- Java version tried: 11 & 17
-  buildToolsVersion = "29.0.3"
- minSdkVersion = 28
 - compileSdkVersion = 33
  -  targetSdkVersion = 33

Error:
```
* What went wrong:
Execution failed for task ':app:mergeDebugAssets'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Failed to transform turbo-local.aar (dev.hotwire:turbo:local) to match attributes {artifactType=android-assets, org.gradle.category=library, org.gradle.dependency.bundling=external, org.gradle.libraryelements=aar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Execution failed for JetifyTransform: /Users/user/.m2/repository/dev/hotwire/turbo/local/turbo-local.aar.
         > Failed to transform '/Users/user/.m2/repository/dev/hotwire/turbo/local/turbo-local.aar' using Jetifier. Reason: IllegalArgumentException, message: Unsupported class file major version 61. (Run with --stacktrace for more details.)
           Suggestions:
            - Check out existing issues at https://issuetracker.google.com/issues?q=componentid:460323&s=modified_time:desc, it's possible that this issue has already been filed there.
            - If this issue has not been filed, please report it at https://issuetracker.google.com/issues/new?component=460323 (run with --stacktrace and provide a stack trace if possible).
   > Failed to transform activity-1.8.1.aar (androidx.activity:activity:1.8.1) to match attributes {artifactType=android-assets, org.gradle.category=library, org.gradle.dependency.bundling=external, org.gradle.libraryelements=aar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Execution failed for JetifyTransform: /Users/user/.gradle/caches/modules-2/files-2.1/androidx.activity/activity/1.8.1/144ff894c8c32584ca9e99ebae0a2a28b294d371/activity-1.8.1.aar.
         > Failed to transform '/Users/user/.gradle/caches/modules-2/files-2.1/androidx.activity/activity/1.8.1/144ff894c8c32584ca9e99ebae0a2a28b294d371/activity-1.8.1.aar' using Jetifier. Reason: IllegalArgumentException, message: Unsupported class file major version 61. (Run with --stacktrace for more details.)
```

### Steps to reproduce

run: `yarn run android`

### Snack or a link to a repository

https://snack.expo.dev/1RYI-XPDdueddj3EpGyCN

### React Native Turbo version

1.0.0

### React Native version

0.67.5

### Platforms

Android

### JavaScript runtime

Hermes

### Workflow

React Native

### Architecture

Paper (Old Architecture)

### Build type

None

### Device

Android emulator

### Device model

Simulator

**Comment by pklatka** [comment]
Hi, I cannot reproduce that issue when setting `minSdkVersion` to 26, `compileSdkVersion` to 34, and upgrading `AGP` to version 7.4.2 (in a bare RN0.67.5 project). Please try to uninstall `react-native-turbo` first, then upgrade your project AGP version, change `minSdkVersion` and `compileSdkVersion`, and finally, install the library.

`react-native-turbo@1.0.0` won't run properly because of [this fancy expression](https://github.com/software-mansion-labs/react-native-turbo-demo/blob/66f1296311f8ea004bf002db03b2f6070920aa48/packages/turbo/src/RNVisitableView.ts#L80), which RN0.67.5 doesn't like, but I'll release 1.0.1, which patches that.

**Comment by pklatka** [comment]
Released `react-native-turbo@1.0.1`.
